lib/logstash/codecs/json_lines.rb

```diff
@@ -28,10 +28,15 @@ class LogStash::Codecs::JSONLines < LogStash::Codecs::Base
   # Change the delimiter that separates lines
   config :delimiter, :validate => :string, :default => "\n"
 
+  # Maximum number of bytes for a single line before a fatal exception is raised
+  # which will stop Logsash.
+  # The default is 20MB which is quite large for a JSON document
+  config :decode_size_limit_bytes, :validate => :number, :default => 20 * (1024 * 1024) # 20MB
```

Comment:
* Is `decode` the right name? The description says this is bytes for a line, but then we call it `decode` which isn't something mentioned elsewhere in the docs.
* Exceeding this will cause a fatal error in Logstash and stop the process? Is this the desired behavior?